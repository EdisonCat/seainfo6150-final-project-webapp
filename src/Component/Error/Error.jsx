import React from "react";
import styles from "./Error.module.css";

const Error = () => {
	return (
		<div className={styles.container}>
			<h1>PAGE NOT FOUND</h1>
			<h2>You've come to nowhere (Knowhere)... =)</h2>
			<blockquote>
				<h3>
					"Knowhere."
					<br />
					"He must be going somewhere."
					<br />
					"No, no. Knowhere? It's a place, we've been there. It
					sucks."
				</h3>
			</blockquote>
			<h4>â€•Thor, Mantis and Star-Lord</h4>
		</div>
	);
};

export default Error;
