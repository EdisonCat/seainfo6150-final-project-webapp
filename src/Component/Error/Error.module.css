@import url('https://fonts.googleapis.com/css2?family=Work+Sans:wght@300&display=swap');
.container {
    display: flex;
    min-height: 900px;
    background: #333;
    border-top: 1px gray solid;
    justify-content: center;
    align-items: center;
    flex-direction: column;
    color: white;
	font-family: 'Work Sans', sans-serif;
}