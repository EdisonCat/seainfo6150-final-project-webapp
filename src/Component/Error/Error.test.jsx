import React from "react";
import Error from "./Error";

it("renders the Error component correctly", () => {
	// eslint-disable-next-line no-undef
  const { container } = render(<Error />);
  expect(container).toMatchSnapshot();
});
