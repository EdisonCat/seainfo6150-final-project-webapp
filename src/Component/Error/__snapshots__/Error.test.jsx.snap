// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Error component correctly 1`] = `
<div>
  <div
    class="container"
  >
    <h1>
      PAGE NOT FOUND
    </h1>
    <h2>
      You've come to nowhere (Knowhere)... =)
    </h2>
    <blockquote>
      <h3>
        "Knowhere."
        <br />
        "He must be going somewhere."
        <br />
        "No, no. Knowhere? It's a place, we've been there. It sucks."
      </h3>
    </blockquote>
    <h4>
      â€•Thor, Mantis and Star-Lord
    </h4>
  </div>
</div>
`;
